### ISSUE TITLE
Blank modlist in linux

### ISSUE THREAD

**Issue by Hope-Caldwell** [RimSort/RimSort#536]
Description:
getting the
"Could not find data for some mods! The following list of mods were set active in your mods list but no data could be found for these mods in local/workshop mod paths.
Are your game configuration paths correct?"
message when trying to open / refresh rimsort, and both the inactive and active mods list being blank

1.  Add / Remove a mod (while rimsort is open in the background?)
2.  Open rimsort

Expected behavior:
RimSort showing the modlists (active and inactive)

Screenshots:
![Screenshot from 2024-07-27 08-55-24](https://github.com/user-attachments/assets/8d946268-db69-419b-80f3-fe49234352f1)
![Screenshot from 2024-07-27 08-55-07](https://github.com/user-attachments/assets/cd83f8ab-f352-488c-9a89-dfbf079b9af3)
![Screenshot from 2024-07-27 08-55-00](https://github.com/user-attachments/assets/e3d72b6e-2b36-4295-ac9a-850132dc5909)

- Operating System: Linux (Pop!_Os based on ubuntu 22.04)
- Processor Architecture: x86_64
- RimSort version: V1.0.9
- Compiled release

The paths look correct.

Log:
[RimSort.log](https://github.com/user-attachments/files/16398374/RimSort.log)

**Comment by hk21702** [comment]
So it looks like there is an error when It's trying to parse your Steam Workshop DB. Specifically, it's trying to open the file but can't because it was provided with a path to a directory instead of a file. Tbh that part does need a rework. Do you mind giving me a screenshot of your Databases settings panel? `File > Settings > Databases`. Please don't hesitate to blot out any identifying information, but I would like to see most of the paths that you have set. 

After getting the screenshots, as a temporary solution just to get things working, play around with the settings there. You can set it to None if you're okay with just disabling the feature, or try to get it to correctly point to the file it wants.

**Comment by Hope-Caldwell** [comment]
turns out it was a user (me) error when setting up locations to the databases
(also, suggestion, can you please add a seperate warning for this)
anyways, thanks for your help and response time

![image](https://github.com/user-attachments/assets/d129174c-895a-4a36-a5fa-5b017e6397c3)

**Comment by hk21702** [comment]
No worries. I'm not going to close this since it sounds like a UX issue that should be addressed at some point.

**Comment by Chunnyluny** [comment]
not to sound too petty but there is a typo in the settings, it says use the 0 to disable the notificatiom M instead of Notification :O I only see it in this screenshot, so I guess a small typo in the linux part then @hk21702

**Comment by hk21702** [comment]
> not to sound too petty but there is a typo in the settings, it says use the 0 to disable the notificatiom M instead of Notification :O I only see it in this screenshot, so I guess a small typo in the linux part then @hk21702

Fixed in #574
